tests/unit/handlers/test_structured_log.py

```diff
@@ -104,6 +104,23 @@ def test_format_minimal(self):
                 value, result[key], f"expected_payload[{key}] != result[{key}]"
             )
 
+    def test_format_with_quotes(self):
+        """
+        When logging a message containing quotes, escape chars should be added
+        """
+        import logging
+        import json
+
+        handler = self._make_one()
+        message = '"test"'
```

Comment:
Are other special chars escaped as well?

Reply:
I haven't seen any that need it. Let me know if you find others, I imagine it will be the same across languages